client/app/data/dashboard/admin.json

```diff
@@ -10,7 +10,7 @@
   {
     "title": "User Access Management",
     "content": "Slightly more to say than what Card A has to say",
-    "links": [{ "title": "Users", "href": "/dashboard/tbd" }]
+    "links": [{ "title": "Users", "href": "/dashboard/access-requests" }]
```

Comment:
I think we're supposed to get to access-requests via an Operator tile: 
![image](https://github.com/bcgov/cas-registration/assets/77306764/7f36316a-b173-43e7-9ee0-c9dfd7ce5c45)
I think someone might have created it already and once you rebase you'll have it